.github/workflows/auto-pr-schemastore.yml

```diff
@@ -0,0 +1,52 @@
+name: Auto PR to Schemastore/schemastore.git
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+     
+jobs:
+  create-auto-PR:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          repository: SchemaStore/schemastore.git
+      
+      - name: Update schema file
+        run: |
+          cd src/schemas/json
+          rm monika-config-schema.json
+          wget https://raw.githubusercontent.com/hyperjumptech/monika/main/monika-config-schema.json
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:         
+          token: ${{ secrets.AUTO-PR-TOKEN }}
+          
+          commit-message: Updating to latest monika-config-schema 
+          committer: GitHub <noreply@github.com>
```

Comment:
Is this GitHub by default or should we use another email instead like monika@hyperjump.tech?

Reply:
We can use any email i think. Does hyperjumptech have public ops/admin email? We can use that.

Reply:
why not just the same like the author `${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>`

Reply:
done.